tests/Parser.test.cpp

```diff
@@ -905,6 +905,87 @@ TEST_CASE_FIXTURE(Fixture, "parse_compound_assignment_error_multiple")
     }
 }
 
+TEST_CASE_FIXTURE(Fixture, "parse_interpolated_string_double_brace_begin")
+{
+    ScopedFastFlag sff{"LuauInterpolatedStringBaseSupport", true};
+
+    try
+    {
+        parse(R"(
+            _ = `{{oops}}`
+        )");
+        FAIL("Expected ParseErrors to be thrown");
+    }
+    catch (const ParseErrors& e)
+    {
+        CHECK_EQ("Double braces are not permitted within interpolated strings. Did you mean '\\{'?", e.getErrors().front().getMessage());
+    }
+}
+
+TEST_CASE_FIXTURE(Fixture, "parse_interpolated_string_double_brace_mid")
+{
+    ScopedFastFlag sff{"LuauInterpolatedStringBaseSupport", true};
+
+    try
+    {
+        parse(R"(
+            _ = `{nice} {{oops}}`
+        )");
+        FAIL("Expected ParseErrors to be thrown");
+    }
+    catch (const ParseErrors& e)
+    {
+        CHECK_EQ("Double braces are not permitted within interpolated strings. Did you mean '\\{'?", e.getErrors().front().getMessage());
+    }
+}
+
+TEST_CASE_FIXTURE(Fixture, "parse_interpolated_string_without_end_brace")
+{
+    ScopedFastFlag sff{"LuauInterpolatedStringBaseSupport", true};
+
+    auto columnOfEndBraceError = [=](const char* code)
```

Comment:
We rarely use `[=]`.
```suggestion
    auto columnOfEndBraceError = [this](const char* code)
```

Reply:
What's the difference? This was my first time learning lambdas.

Reply:
It's mostly an internal style guide to _prefer_ explicit captures, up to a degree. The only thing this lambda uses is `this`. This is really the main reason why I asked for this change.

With `[=]`, anything captured is copied by value, whereas with `[&]` anything captured retains a reference to the lvalue.

The difference in this case is nothing, since `this` is a pointer to `Fixture`, so it just copies the pointer address itself to the Fixture.